### DISCUSSION TITLE
Find 3 singular nouns which ...

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#834] [Galaxy Trivia] [answered]
... contain EE twice and are at least 9 characters long.

**Comment by peterbaccaro** [comment, marked as answer]
Three singular nouns that have EE twice and are at least 9 characters long are:

- **Beekeeper**
- **Greenkeeper**
- **Beekeeping**

**Reply by ParanoidUser** [reply]
**Here is another one:** Why did the bicycle refuse to go on a diet? Because it wanted to maintain its *freewheel* figure!

**Comment by hasanerzi** [comment]
here is:

1. Fleetingly
2. Sightseeing
3. Engineering

**Reply by ParanoidUser** [reply]
Nice try, but I'm afraid there's only one "EE" combination out there. 🫣

**Comment by anjutares** [comment]
Here are three examples of singular nouns that contain the letter combination "ee" twice and are at least 9 characters long:

    Committee
    Licensee
    Guarantee
